website/docs/layouts/app-frame/partials/code/how-to-use.md

```diff
@@ -124,3 +124,20 @@ If the “modal“ container is empty, a `display: none` style is applied to it.
   </Hds::AppFrame>
 </div>
 ```
+
+If the content is injected dynamically –eg. via JavaScript or via Ember "portals"– you can assign an ID to the HTML element so that it can be targeted in the DOM by the code:
```

Comment:
```suggestion
If the content is injected dynamically—eg. via JavaScript or via Ember "portals"—you can assign an ID to the HTML element so that it can be targeted in the DOM by the code:
```